src/test/kotlin/pt/up/fe/ni/website/backend/permissions/PermissionsTest.kt

```diff
@@ -0,0 +1,12 @@
+package pt.up.fe.ni.website.backend.permissions
+
+import org.junit.jupiter.api.Test
+
+class PermissionsTest {
+
+    @Test
+    fun permissionsShouldBeBuilt() {
```

Comment:
I didn't quite understand the test, can you explain?

Reply:
Nevermind that, I wrote it for debugging things and forgot to remove it!